src/libraries/System.Private.CoreLib/src/System/OperatingSystem.cs

```diff
@@ -65,7 +65,7 @@ public string VersionString
                         case PlatformID.Unix: os = "Unix "; break;
                         case PlatformID.Xbox: os = "Xbox "; break;
                         case PlatformID.MacOSX: os = "Mac OS X "; break;
-                        case PlatformID.Other: os = "Other"; break;
+                        case PlatformID.Other: os = "Other "; break;
```

Comment:
What test should have caught this?

Reply:
We don't have one currently since we only do a `.Contains()` here without the space: https://github.com/dotnet/runtime/blob/a92304e2124640f3c8b5d3b0bd8fb35e539cc4dc/src/libraries/System.Runtime.Extensions/tests/System/EnvironmentTests.cs#L145-L146

Should we change that test or add an explicit new one? I don't have a strong opinion since this is a string used mostly for display and `Other` should be the last time we touch this piece.

Reply:
Let's just update the existing test.  Thanks.

> Other should be the last time we touch this piece.

Famous last words ;-)

Reply:
Updated the test ;)